blobscache/client.go

```diff
@@ -42,7 +42,7 @@ func (c *remoteBlobsCache) PutBlob(ctx context.Context, key string, blob []byte)
 	}); err != nil {
 		return err
 	}
-	req, err := http.NewRequestWithContext(ctx, "POST", fmt.Sprintf("%s/PutBlob", c.url), &buf)
+	req, err := http.NewRequestWithContext(ctx, "POST", fmt.Sprintf("%s/v1/blobscache/PutBlob", c.url), &buf)
```

Comment:
Could you point me to documentation or source code where `/v1/blobscache` is defined? I was not able to find it.